cart.html

```diff
@@ -1,90 +1,82 @@
-                </ul>
-                <!-- Icons for for the user and cart -->
-                <ul class="login-cart">  
-                    <!--Login-->
-                    <li>
-                        <a href="login.html"><i class="fas fa-user"></i></a>
-                    </li>  
-                    <!--Cart-->
-                    <li>
-                        <a href="cart.html"
-                            ><i class="fas fa-shopping-cart"></i
-                        ></a>
-                    </li>
-                </ul>
-            </nav>
+            <div class="text-center mt-5">
+                <h1 class="text-dark icon-collapse">Your Cart</h1>
+            </div>
         </header>
 
-        <section>
-            <header>
-                <div class="text-center mt-5">
-                    <h1 class="text-dark icon-collapse">Your Cart</h1>
-                </div>
-            </header>
+        <article class="mt-5"></article>
+    </section>
 
-            <article class="mt-5"></article>
-        </section>
+    <footer>
+        <h5>Planet wine</h5>
+        <p>3700 Willingdon Ave, Burnaby, BC V5G 3H2</p>
+        <div class="social">
+            <ul>
+                <li>
+                    <a href="#"><i class="fab fa-facebook"></i></a>
+                </li>
+                <li>
+                    <a href="#"><i class="fab fa-twitter"></i></a>
+                </li>
+                <li>
+                    <a href="#"><i class="fab fa-instagram"></i></a>
+                </li>
+                <li>
+                    <a href="#"><i class="fab fa-youtube"></i></a>
+                </li>
+            </ul>
+        </div>
+        <p>&copy; All Right Reserved</p>
+    </footer>
 
-        <footer>
-            <h5>Planet wine</h5>
-            <p>3700 Willingdon Ave, Burnaby, BC V5G 3H2</p>
-            <div class="social">
-                <ul>
-                    <li>
-                        <a href="#"><i class="fab fa-facebook"></i></a>
-                    </li>
-                    <li>
-                        <a href="#"><i class="fab fa-twitter"></i></a>
-                    </li>
-                    <li>
-                        <a href="#"><i class="fab fa-instagram"></i></a>
-                    </li>
-                    <li>
-                        <a href="#"><i class="fab fa-youtube"></i></a>
-                    </li>
-                </ul>
-            </div>
-            <p>&copy; All Right Reserved</p>
-        </footer>
+    <!-- <script src="scripts/index.js"></script> -->
+    <script src="scripts/index.js"></script>
+    <script src="scripts/cart.js"></script>
+    <script src="scripts/global-toggle.js"></script>
+</body>
 
-        <!-- <script src="scripts/index.js"></script> -->
-        <script src="scripts/index.js"></script>
-        <script src="scripts/cart.js"></script>
-        <script src="scripts/global-toggle.js"></script>
-    </body>
-</html>
+</html>
```

Comment:
For this file it seems like you just un-tab everything in the html tag which some people do.
For consistency between pages though it is better to stick with one style.
Would probably be fine though, Steven can just make every page consistent by running through them all with prettier.